File: src/Api/Logic/Models/RetreatStatistics.cs
namespace Api.Logic.Models;

public class RetreatStatistics
{
    public Guid Id { get; set; }
    public string PlayerOneName { get; set; }
    public string PlayerTwoName { get; set; }
    public DateTime StartDateTime { get; set; }
    public DateTime EndDateTime { get; set; }
    public int MouseDistance { get; set; }
    public int ClicksCount { get; set; }
    public int ShortCutsCount { get; set; }
}

File: src/front/contracts/RetreatStatistics.ts
// tslint:disable
// TypeScriptContractGenerator's generated content
import { Guid } from './../domain/dataTypes/Guid';
import { DateTime } from './../domain/dataTypes/DateTime';

export type RetreatStatistics = {
    id: Guid;
    playerOneName: Nullable<string>;
    playerTwoName: Nullable<string>;
    startDateTime: DateTime;
    endDateTime: DateTime;
    mouseDistance: number;
    clicksCount: number;
    shortCutsCount: number;
};
